contracts/src/bridge/frontend/Dockerfile

```diff
@@ -17,11 +19,6 @@ ENV NEXT_PUBLIC_BRIDGE_L2=${L2_BRIDGE}
 ENV NEXT_PUBLIC_BRIDGE_MESSAGE_BUS=${MESSAGE_BUS}
 ENV NEXT_PUBLIC_BRIDGE_GOOGLE_ANALYTICS_ID=${GOOGLE_ANALYTICS_ID}
 
-# Copy package.json and package-lock.json (or yarn.lock) into the container
-COPY package*.json ./
-
 RUN npm install
-COPY . .
-RUN npm run build
-EXPOSE 80
-CMD ["npm", "start"]
\ No newline at end of file
+
+EXPOSE 80
```

Comment:
Not sure if you still need `CMD ["npm", "start"]` here? Depends if that's getting called by whatever kicks off the server I guess, maybe you don't need it if you're copying how it works for some other app